src/main/resources/static/js/MainWindow.js

```diff
@@ -489,7 +490,8 @@ function resetConfig() {
     $("#configs").val(null).trigger("change");
     resetMapping();
 
-    optimzeforsource(); //disables the delimiter and time format if the source file is an excel file
+    optimzeforsource(); // disables the delimiter and time format if the source
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['optimzeforsource' is not defined.](https://app.codacy.com/gh/FraunhoferIOSB/ChillImport/pullRequest?prid=4781409)